### PR TITLE
Feat: treemap/zoom

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
[09/18] 리뷰완료
고생하셨습니다!

**Review by jungeun-cho — APPROVED** [review]
[9/18] 이견없습니다. 수고하셨어요.

File: src/component/treemapSeries.ts [hunk 1]
```diff
@@ -161,6 +192,18 @@ export default class TreemapSeries extends Component {
...
   }
 
+  onClick({ responders }) {
+    if (this.zoomable && responders[0]) {
+      const { id, hasChild } = responders[0];
+
+      if (hasChild) {
+        this.emitMouseEvent([]);
+        this.store.dispatch('setTreemapZoomId', id);
+        this.eventBus.emit('needDraw');
```
Comment (shiren) [inline src/component/treemapSeries.ts:202]: 여기도 needDraw를 수동으로해야해요? dispatch가 있으니 기본 사이클을 탈것 같아서요

**Review by shiren — APPROVED** [review]
수고하셨어요

File: src/component/treemapSeries.ts (hunk 1 above)
Comment (jung-han) [reply src/component/treemapSeries.ts:203]: 제거해주겠습니다!

**Comment by jung-han** [comment]
ok to test

**Pull request merged**